### PR TITLE
Add translation for teams page

### PR DESCRIPTION
**Pull request by Flip-HH**
- Added i18n definitions for english and french
- Replaced team member titles and programs using the new translation objects

Don't hesitate to comment if there are typos :/

### CODE REVIEW

File: src/plugins/i18n.ts [hunk 1]
```diff
@@ -190,6 +222,34 @@ const rawMessages: Record<string, Record<Lang, string>> = {
     en: 'Software',
     fr: 'Logiciel',
   },
+  title_captain: {
+    en: 'Captian',
```
Comment (McCaroon) [inline src/plugins/i18n.ts:226]: Typo: captain *

Comment (Flip-HH) [reply src/plugins/i18n.ts]: Fixed

**Comment by McCaroon** [comment]
Function-wise, it looks good.
Text-wise, it looks good.
I'd just wait to see if @Leuchak has anything relevant to point out before merging, since web design isn't my forte. Good job! 👍

File: src/plugins/i18n.ts (hunk 1 above)
Comment (AntoineMaltais) [reply src/plugins/i18n.ts]: bro didnt push his code

**Review by AntoineMaltais — APPROVED** [review]
lgtm(jai aussi tester)

**Review by AntoineMaltais — APPROVED** [review]
lgtm(j'ai tester aussi)

File: src/views/TeamView.vue [hunk 2]
```diff
@@ -227,8 +227,10 @@ const sections = [
               :key="j"
               :name="member.name"
               :img="member.img ?? defaultAvatar"
-              :program="member.program"
-              :title="member.title ?? $t('team_member')"
+              :program="$t(member.program)"
+              :title="
+                member.title == undefined ? $t('team_member') : $t(member.title)
+              "
```
Comment (AntoineMaltais) [inline src/views/TeamView.vue:233]: revert this change its does the same thing and is longer

**Review by AntoineMaltais — CHANGES_REQUESTED** [review]

**Review by AntoineMaltais — APPROVED** [review]

**Review by AntoineMaltais — APPROVED** [review]

**Review by Leuchak — CHANGES_REQUESTED** [review]
As he said

**Comment by Flip-HH** [comment]
> As he said

The change is necessary in this case, because the old method was checking if the string was undefined, then printing the other one. In the case where it is not undefined, it is simply printed to the html page.

When using the translation object, we need to check that the string is undefined before putting it in the object. If i simply added the $t around the string and left it as it was, the translation object tries to fetch a translation string based on the locale but since it's undefined, it doesn't work, which is why i changed the method to check that the string is undefined before doing the outputs.

**Review by Leuchak — APPROVED** [review]

**Comment by Leuchak** [comment]
good

**Pull request merged**